sidechain/consensus/aura/src/block_production_suspension.rs

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
a bit late, but I only noticed now : Why are you placing the block production suspender in aura and not in common folder? I don't see any aura specifc features here.

(same for the peer block syncer)

Reply:
Good catch. I second that question.

Reply:
I can move those, they are both not aura specific 👍